practice-app/app/views.py

```diff
@@ -1,15 +1,99 @@
-from flask import Flask, render_template, url_for, request
+from flask import Flask, render_template, url_for, request, redirect
 import requests
+from flask_wtf import FlaskForm
+from flask_bootstrap import Bootstrap
+from wtforms import StringField, PasswordField, BooleanField
+from wtforms.validators import InputRequired, Length
+from sqlalchemy import create_engine, Column, Integer, String
+from sqlalchemy.orm import sessionmaker, scoped_session
+from sqlalchemy.ext.declarative import declarative_base
+from flask_login import LoginManager, UserMixin, login_user, login_required, logout_user
 
-app = Flask(__name__)   # name is global variable returning program name which is app
+engine = create_engine('postgresql://postgres:Practice1.@localhost:5432/postgres', echo=False)
```

Comment:
I think it would be better to get username, password and database name from environment. In current way we need to change these everytime but if we put them in environment and pull from there we dont need to do any change.

Reply:
you can see the second commit on this pull request